src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Component.targets

```diff
@@ -0,0 +1,302 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.Razor.Component.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project ToolsVersion="14.0">
+
+  <PropertyGroup>
+    <!-- Used for tracking inputs to RazorGenerateDeclarations -->
+    <_RazorComponentInputHash></_RazorComponentInputHash>
+    <_RazorComponentInputHashFile>$(IntermediateOutputPath)$(TargetName).RazorComponent.input.cache</_RazorComponentInputHashFile>
+
+    <_RazorComponentDeclarationManifest>$(IntermediateOutputPath)RazorComponents.declaration.json</_RazorComponentDeclarationManifest>
+    <_RazorComponentDefinitionManifest>$(IntermediateOutputPath)RazorComponents.definition.json</_RazorComponentDefinitionManifest>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Path used for the temporary compilation we produce for component discovery -->
+    <_RazorComponentDeclarationAssembly Include="$(IntermediateOutputPath)RazorTemp\$(TargetName).dll" />
+  </ItemGroup>
+
+  <Target Name="_ResolveRazorDeclarationConfiguration">
+    <PropertyGroup>
+      <_ResolvedRazorComponentDeclarationConfiguration>@(ResolvedRazorConfiguration->Metadata('DeclarationConfiguration'))</_ResolvedRazorComponentDeclarationConfiguration>
```

Comment:
I wonder if we could make this a first class option rather than a separate configuration.